src/hetzner/instance/create.cr

```diff
@@ -212,6 +212,8 @@ class Hetzner::Instance::Create
     [
       "hostnamectl set-hostname $(curl http://169.254.169.254/hetzner/v1/metadata/hostname)",
       "update-crypto-policies --set DEFAULT:SHA1 || true",
+      "systemctl disable --now ssh.socket",
+      "systemctl enable --now ssh.service",
```

Comment:
Can you change this slightly for my so these commands fail gracefully if not supported by the OS?

```suggestion
      "systemctl disable --now ssh.socket || true",
      "systemctl enable --now ssh.service || true",
```

Do you see any negative implications with this?

Reply:
Good idea! I made the change.